### ISSUE TITLE
no asset for chart.js file

### ISSUE THREAD

**Issue by nicdnepr** [Practically/yii2-chartjs#25]
Why no asset file for publish chart.js file? it's not working extension

**Comment by paskuale75** [comment]
You should load the library into the assets via url (cdn)
I paste here my code:
```
class BudgetStatsAsset extends AssetBundle
{
    public $sourcePath = '@app/widgets/....../assets';

   //the last useful version, from version 4 it is no longer compatible
    public $js = [
        'https://cdnjs.cloudflare.com/ajax/libs/Chart.js/3.9.1/chart.js' 
    ];

    public $css = [
    ];

    public $depends = [
        'yii\web\JqueryAsset'
    ];

    public function init()
    {
        // Tell AssetBundle where the assets files are
        $this->sourcePath = __DIR__ . "/assets";
        parent::init();
    }
}
```

**Comment by gwynbox** [comment]
resolved in v1.3.0
